project/src/test/java/com/google/sps/servlets/QueryServletTest.java

```diff
@@ -95,8 +112,20 @@ public void getRequest_fetchedLessThanMaxNumPlaces_respondAllFetchedPlaces() thr
 
   // Returns the number of json elements in the servlet's response
   private int getPlacesAmountInResponse() {
+    System.out.println(responseStringWriter.getBuffer());
+    System.out.println(responseStringWriter.getBuffer().toString());
+    System.out.println(new Gson()
+    .fromJson(responseStringWriter.getBuffer().toString(), JsonArray.class));
     return new Gson()
         .fromJson(responseStringWriter.getBuffer().toString(), JsonArray.class)
         .size();
   }
+
+  private void setRequestParameters(String quisines) {
```

Comment:
I like this solution of putting all the getPramater treatment in one function :)

Reply:
Thank you!